.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
```

Comment:
Is this file specific to your own text editor?

Reply:
It’s a config file to standardize the format of all our files. I know a few text editors use this plugin and we have it in our cvs repo so I figured it was a good thing to add to this project too.